test/tests/functional/pbs_equiv_classes.py

```diff
@@ -1931,3 +1931,31 @@ def test_overlap_resv(self):
 
         self.server.expect(JOB, {"job_state": 'R'}, id=jid1)
         self.server.expect(JOB, {"job_state": 'R'}, id=jid3)
+
+    def test_limit_res(self):
+        """
+        Test when resources are being limited on, but those resources are not
+        in the sched_config resources line.  Jobs requesting these resources
+        should be split into their own equivalence classes.
+        """
+        a = {ATTR_RESC_TYPE: 'long'}
+        self.server.manager(MGR_CMD_CREATE, RSC, a, id='foores')
+
+        a = {'max_run_res.foores': '[u:PBS_GENERIC=4]'}
+        self.server.manager(MGR_CMD_SET, SERVER, a)
+
+        a = {'Resource_List.foores': 1, 'Resource_List.select': '1:ncpus=1'}
+        self.submit_jobs(2, a)
+        a['Resource_List.foores'] = 2
+        (_, jid4) = self.submit_jobs(2, a)
```

Comment:
Parentheses are not necessary here.

Reply:
Oh, wow, you're right.  I thought because submit_jobs() returns a list that I needed the ().  In any case, the entire test plan uses the ().  I'll either leave them alone, or pull them from the entire plan, and I'd rather not pull them from the entire plan.

Reply:
That's fine. It just surprised me a little bit because parentheses makes it looks like `submit_jobs` always returns a tuple instead of a list. I had to go to the definition to check.